dmt/main/models.py

```diff
@@ -710,6 +710,31 @@ def all_actual_times(self):
             item__milestone__project=self,
         ).order_by('completed')
 
+    def users_active_between(self, start, end):
```

Comment:
imho, the raw sql is ok if required for performance. The Django ORM aggregation functions are worth a look if you haven't already discounted them. https://docs.djangoproject.com/en/1.7/topics/db/aggregation/